src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
```

Comment:
The question says all animals start with 100 health, so there's no need to receive health in the constructor, you could simply fix it to 100, just like you did for the `hidden` property for the Herbivore.
But this is ok, as it gives the code more flexibility if future implementations ask for it